p8e-sdk/src/main/resources/version.properties

```diff
@@ -0,0 +1 @@
+version=${version}
```

Comment:
Does the semver plugin manage setting this on builds or something?

Reply:
this block in the gradle file populates the version number https://github.com/provenance-io/p8e/pull/100/files#diff-49a96e7eea8a94af862798a45174e6ac43eb4f8b4bd40759b5da63ba31ec3ef7R67

I was hoping there was some more automatic way to pull the version in, but apparently not

Reply:
the semver plugin is just to assist in comparing semver strings to determine if it is a newer or older version

Reply:
Ah got it.